plasma_framework/contracts/src/exits/payment/PaymentInFlightExitable.sol

```diff
@@ -85,9 +92,21 @@ contract PaymentInFlightExitable is
         uint256[] inputUtxosTypes;
         bytes[] inputTxsInclusionProofs;
         bytes[] inFlightTxWitnesses;
+        // TODO: Output Ids are only computed and not used in exit
```

Comment:
@pdobacz @boolafish: After discussed changes in `IPaymentSpendingCondition` output ids seems to be not used...
also this exit game is for Payment Tx - so for now it's using `utxoPos`-based input identifiers.

Should I live it as is - or remove?

Reply:
oh ouch. Now I remembered why I put both `outputId` and `utxoPos` in the interface and passing in both valid value from the ExitGame contract to Spending Condition. I did this to make sure in future we don't need to break into ExitGame and change it from passing `utxoPos` to passing `outputId` so hopefully we don't need to re-audit the ExitGame contract when doing upgrade. ah, hope #214 would solve this.


Back to your question. With current interface, if you're not using it then it should be fine to not introduce that. Meanwhile, it should be separated as outputIds for inputs and outputs. We need to do a final check of all inputs' outputIds but flag both in/output outputIds as spent on `processExit`.